services/auth_service.py

```
@@ -47,24 +48,36 @@ def signup_user(data: UserSignUp):
 
-    if not user or not verify_password(data.password, user["password"]):
-        raise HTTPException(401, "인증 실패")
+            if not verify_password(data.password, user_record["password"]):
+                raise HTTPException(401, "인증 실패")
 
-    return user, *create_access_token(user["user_id"])
+            token, expires = create_access_token(user_record["user_id"])
+            return user_record, token, expires
+    except Exception as ex:
+        con.rollback()
+        raise ex
```

Ruff (0.14.13)

Use `raise` without specifying exception name

Remove exception name

(TRY201)